.github/actions/go-aio/action.yaml

```
@@ -12,15 +12,20 @@ inputs:
     default: ""
   version:
     description: "Go version"
-    required: true
+    required: false
     default: "1.18"
+  version-file:
+    description: "Go version file (go.mod)"
+    required: false
+    default: ""
```

```suggestion
    default: "go.mod"
```
?
From https://github.com/actions/setup-go
> If both the go-version and the go-version-file inputs are provided then the go-version input is used.